src/vl-wizard-pane.js

```diff
@@ -32,6 +32,10 @@ export class VlWizardPane extends VlElement(HTMLElement) {
                 slot[name="previous-action"], slot[name="next-action"] {
                     display: inline-block;
                 }
+
+                [hidden] {
+                    display: none !important;
+                }
```

Comment:
Waarom deze ruleset er bij?

Reply:
Stijl schoonheidsfoutje opgelost dat ik in deze PR toevallig gezien heb.